digital-thoreau-simplify/usr/share/asteroid-launcher/watchfaces/digital-thoreau-simplify.qml

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (C) 2021 - Ed Beroset <github.com/beroset>
+ * All rights reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 2.1 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import QtQuick 2.12
+import QtGraphicalEffects 1.12
+
+Item {
+
+    Image {
+        z: 1
+        id: asteroidLogo
+        visible: !displayAmbient
+        source: "../watchfaces-img/asteroid-logo.svg"
+        antialiasing: true
+        anchors {
+            centerIn: parent
+            verticalCenterOffset: -parent.height * 0.272
+        }
+        width: parent.width * 0.12
+        height: parent.height * 0.12
+        opacity: 0.7
+
+        Text {
+            z: 2
+            id: asteroidSlogan
+            visible: !displayAmbient
+            font.pixelSize: parent.height * 0.31
+            font.family: "Raleway"
+            color: "white"
+            horizontalAlignment: Text.AlignHCenter
+            anchors {
+                centerIn: parent
+                verticalCenterOffset: -parent.height * 0.005
+            }
+            text: "<b>AsteroidOS</b><br>Hack Your Wrist"
+        }
+        MouseArea {
+            anchors.fill: parent
+            onPressAndHold: asteroidLogo.visible ?
+                                asteroidLogo.visible = false :
+                                asteroidLogo.visible = true
+        }
+    }
+
+    Text {
+        id: datetext
+        anchors {
+            centerIn: parent
+            verticalCenterOffset: parent.height * 0.25 
+        }
+        font.family: "Titillium"
+        font.weight: Font.Thin
+        font.pixelSize: parent.height * 0.06
+        color: "white"
+        text: wallClock.time.toLocaleString(Qt.locale(), "ddd dd");
+    }
+
+    Text {
+        id: time
+        anchors.centerIn: parent
+        font.family: "Titillium"
+        font.weight: Font.Thin
+        font.pixelSize: parent.height * 0.30
+        color: "white"
+        text: wallClock.time.toLocaleString(Qt.locale(), (use12H.value ? "hhmm ap" : "HHmm")).slice(0,4);
+    }
+
+    Text {
+        id: ampm
+        visible: use12H.value
+        anchors {
+            centerIn: parent
+            verticalCenterOffset: parent.height * 0.143
+        }
+        font.family: "Titillium"
```

Comment:
When ever more than two properties of a type are declared, i find the 
```
font {
  property: n
}
```
syntax much better readable. It gives me better oversight (from beginner pov).
And one could go into homeopathic code duplication by saving some chars. But i will not :D

Reply:
Another good comment.  I also changed it so that the font family is always named first everywhere.